OpenUpSA/wazimap-ng-ui
# Format and show snackbar on datamapper filter changes

## Description
Added first react component for mapchip

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-400

## How is it tested automatically?

## How should a reviewer test it locally
* Select any indicator on datamapper
* Change Filters
* Select any another indicator
* Snackbar should appear above mapchip regardless of mapchip is expanded or not

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
